packages/apple-pay-integration/src/mocks/apple-pay-button.mock.ts

```diff
@@ -1,7 +1,32 @@
-import { CheckoutButtonInitializeOptions } from '@bigcommerce/checkout-sdk/payment-integration-api';
+import {
+    BuyNowCartRequestBody,
+    CartSource,
+    CheckoutButtonInitializeOptions,
+} from '@bigcommerce/checkout-sdk/payment-integration-api';
 
 import { WithApplePayButtonInitializeOptions } from '../apple-pay-button-initialize-options';
 import ApplePayButtonMethodType from '../apple-pay-button-method-type';
+import { getCart } from '@bigcommerce/checkout-sdk/payment-integrations-test-utils';
+
+const buyNowCartRequestBody: BuyNowCartRequestBody = {
+    source: CartSource.BuyNow,
+    lineItems: [
+        {
+            productId: 1,
+            quantity: 2,
+            optionSelections: {
+                optionId: 11,
+                optionValue: 11,
+            },
+        },
+    ],
+};
+
+export const buyNowCartMock = {
+    ...getCart(),
+    id: 999,
+    source: CartSource.BuyNow,
+};
```

Comment:
You can use getBuyNowCart from @bigcommerce/checkout-sdk/payment-integrations-test-utils

Reply:
You can remove this code and use getBuyNowCart method in apple pay spec file

Reply:
fixed